releasenotes/notes/deprecate-qinfo-synthesis-23abd8c34fc0f52e.yaml

```
@@ -0,0 +1,24 @@
+---
+deprecations:
+  - |
+    The :mod:`qiskit.quantum_info.synthesis` module is pending deprecation and
+    will be deprecated in a future release. The following objects have been moved to
+    :mod:`qiskit.synthesis`:
+
+    * :class:`~.OneQubitEulerDecomposer` has been moved to :mod:`qiskit.synthesis.one_qubit`
+    * :class:`~.TwoQubitBasisDecomposer` has been moved to :mod:`qiskit.synthesis.two_qubits`
+    * :class:`~.XXDecomposer` has been moved to :mod:`qiskit.synthesis.two_qubits`
+    * :func:`~.two_qubit_cnot_decompose` has been moved to :mod:`qiskit.synthesis.two_qubits`
+    * :func:`~.qs_decomposition` has been moved to :mod:`qiskit.synthesis.unitary`
+
+    The class :class:`.Quaternion` has been migrated from :mod:`qiskit.quantum_info.synthesis` to
+    :mod:`qiskit.quantum_info`. This move has not affected the usual import path of the class.
+
+    This function is pending deprecation and will be deprecated in a next release:
+
+    * :func:`~.cnot_rxx_decompose`
+
+    These functions were removed, since they have already been deprecated in a previous release:
+
+    * :func:`~.decompose_clifford` (use :func:`.qiskit.synthesis.synth_clifford_full` instead)
+    * :func:`~.decompose_cnotdihedral` (use :func:`.qiskit.synthesis.synth_cnotdihedral_full` instead)
```

The removals should be in an `upgrade:` section in the releasenote. Since they've been removed, we can also no longer link to them 🙂

---

we still need to remove the links if they are removed 🙂 
```suggestion
    * ``decompose_clifford`` (use :func:`.qiskit.synthesis.synth_clifford_full` instead)
    * ``decompose_cnotdihedral`` (use :func:`.qiskit.synthesis.synth_cnotdihedral_full` instead)
```